src/rgw/rgw_op.cc

```diff
@@ -3810,9 +3810,12 @@ void RGWPutObj::execute(optional_yield y)
   // make reservation for notification if needed
   std::unique_ptr<rgw::sal::Notification> res = store->get_notification(s->object.get(),
 						s, rgw::notify::ObjectCreatedPut);
-  op_ret = res->publish_reserve(this, obj_tags.get());
-  if (op_ret < 0) {
-    return;
+  if(! multipart)
+  {
```

Comment:
```suggestion
  if (!multipart) {
```

Reply:
@TRYTOBE8TME please fix the above, no need to run teuthology again after the fix.